pkgs/data/themes/mcos-dark-xfce-edition/default.nix

```diff
@@ -0,0 +1,28 @@
+{ stdenv, fetchFromGitHub, gtk-engine-murrine }:
+
+stdenv.mkDerivation rec {
+  name = "mcos-dark-xfce-edition";
```

Comment:
It is missing a version number. Maybe the date of the revision should be used for that.

Reply:
Also the base name is `McOS-MJV-Dark-XFCE-Edition`. Your name is missing the `MJV` part.

Reply:
The author has other OSX themes besides Mojave (MJV), like High Sierra (HS) and Yosemite (YS).

Reply:
Done!